scripts/globals/items/scroll_of_instant_shell.lua

```diff
@@ -13,20 +12,10 @@ item_object.onItemCheck = function(target)
 end
 
 item_object.onItemUse = function(target)
-    local power = 1
+    local power = 2188 -- shell_iii base power
```

Comment:
Like I was saying on discord, bg says it uses the old pre-adjustment shell 3 power. Shell 3 used to be 19% (48/255) before being moved to its modern 22% (56/255)

when SE updated the spells, they left items alone (which is why the protect scroll is giving 75 and not the 90 that prot 3 does now)

Reply:
i understand now, sorry bout that

Reply:
so if the power is base 10000, 19% would be 1900, correct? :)